### PR TITLE
Add `TagIgnore`  & `ProfileName` to Config File

### LINKED ISSUE #3: [Feat][upgradinatorr] Ignore tags
It would be useful (especially for sonarr integration) to be able to set tags to ignore either via ID or name. 

Ex.
Anime for example is for me at least done by hand, (for older anime especially), or kids shows are done by hand as well (what a nightmare most of them are).

### LINKED ISSUE #10: [Feat][Upgradinatorr] Search by Profile
Kind of along the lines of ignore by tag you could also add a `--profile " "` tag and the script will only search that profiles

### PR DESCRIPTION
**Pull request by angrycuban13**
- [ ] Script refactoring
  - [ ] Radarr:
    - [ ] Create function to retrieve `tagIgnore` tags
    - [x] Create function to retrieve `profileName` profile ID
    - [ ] Create logic to filter movies based on if `tagIgnore` is filled in
    - [x] Create logic to filter movies based on if `profileName` is filled in
    - [ ] Create logic to filter movies based on if `tagIgnore` and `profileName` are filled in
    - [x] Create logic to filter movies if neither of the two are filled in
  - [ ] Sonarr:
    - [ ] Create function to retrieve `tagIgnore` tags
    - [ ] Create function to retrieve `profileName` profile ID
      - [ ] Test `Get-ProfileQuality` function for Sonarr 
    - [ ] Create logic to filter series based on if `tagIgnore` is filled in
    - [ ] Create logic to filter series based on if `profileName` is filled in
    - [ ] Create logic to filter series based on if `tagIgnore` and `profileName` are filled in
    - [ ] Create logic to filter series if neither of the two are filled in
  - [ ] Continue cleaning up all the shit Baker did
- [ ] Documentation
- [ ] Testing

Closes #3
Closes #10

### CODE REVIEW

**Comment by angrycuban13** [comment]
> **PSScriptAnalyzer results:**
> 
> Errors: [0], Warnings: [5], Information: [2]

@bakerboy448  just for you

File: Upgradinatorr/upgradinatorr.ps1 [hunk 1]
```diff
@@ -389,23 +420,63 @@ foreach ($app in $apps) {
...
     Write-Output "Config - $((Get-Culture).TextInfo.ToTitleCase($app)) Limit: $($count)"
+    switch ($profileName){
+        $null {
+            Write-Warning "Config - Profile: No profile specified"
+        }
+        {$profileName.count -eq 1} {
+            Write-Output "Config - Profile: $profileName"
+        }
+    }
+    if ($profileName.count -gt 1){
```
Comment (bakerboy448) [inline Upgradinatorr/upgradinatorr.ps1:434]: Move this inside the switch

File: Upgradinatorr/upgradinatorr.ps1 [hunk 2]
```diff
@@ -389,23 +420,63 @@ foreach ($app in $apps) {
...
+    Write-Output "Config - Tag name: $($tagName)"
+    switch ($tagIgnore){
+        $null {
+            Write-Warning "Config - Tag ignore list: No tag blacklist specified"
+        }
+        {$tagIgnore.count -eq 1} {
+            Write-Output "Config - Tag ignore list: $tagIgnore"
+        }
+    }
+    if ($tagIgnore.count -gt 1){
```
Comment (bakerboy448) [inline Upgradinatorr/upgradinatorr.ps1:446]: Also move within the switch

File: Upgradinatorr/upgradinatorr.ps1 (hunk 1 above)
Comment (angrycuban13) [reply Upgradinatorr/upgradinatorr.ps1:434]: Does the commit not say `WIP`?

File: Upgradinatorr/upgradinatorr.ps1 (hunk 2 above)
Comment (angrycuban13) [reply Upgradinatorr/upgradinatorr.ps1:446]: Does the commit not say `WIP`?

**Pull request closed without merging**